experimental/generation/generator/packages/library/templates/standard/form-BeginDialog.dialog.lg

```diff
@@ -30,9 +30,9 @@
 ```
 
 # setPropertiesForDynamiclists
-- IF: ${!empty(where(schema.properties, property, property.value.$template == "dynamiclist"))}
+- IF: ${!empty(where(schema.properties, property, property.value.$template == "dynamiclist" || property.value.$template == "LuceneDynamicList" ))}
```

Comment:
> || property.value.$template == "LuceneDynamicList"  [](http://example.com/codeflow?start=91&length=52)

You can't make this change--it presumes knowledge that these templates don't have.  You could revamp this to use a transformer to add the set properties and use the transformer as part of the lucene templates.

Reply:
You still can't make this change--revert it.